.github/workflows/test-runner.yml

```
@@ -19,7 +19,7 @@ jobs:
     strategy:
       matrix:
         python-version: [3.8, 3.9]
```

@jonhealy1 I think support for 3.7 should be dropped (another comment), and gitlab ci should also test 3.10 and 3.11

```suggestion
        python-version: ["3.8", "3.9", "3.10", "3.11"]
```